src/Explorer/Panes/SettingsPane/index.tsx

```diff
@@ -0,0 +1,408 @@
+import { PrimaryButton } from "office-ui-fabric-react";
+import React, { FunctionComponent, KeyboardEvent, useEffect, useLayoutEffect, useRef, useState } from "react";
+import * as Constants from "../../../Common/Constants";
+import { configContext } from "../../../ConfigContext";
+import { LocalStorageUtility, StorageKey } from "../../../Shared/StorageUtility";
+import * as StringUtility from "../../../Shared/StringUtility";
+import { userContext } from "../../../UserContext";
+import * as NotificationConsoleUtils from "../../../Utils/NotificationConsoleUtils";
+import { Tooltip } from "./Tooltip";
+
+export interface SettingsPaneProps {
+  closePanel: () => void;
+  openNotificationConsole: () => void;
+}
+
+export const SettingsPane: FunctionComponent<SettingsPaneProps> = ({
+  closePanel,
+  openNotificationConsole,
+}: SettingsPaneProps) => {
+  const [formErrors, setFormErrors] = useState("");
```

Comment:
Still need type parameters in this file. Example `useState<string>("")`

Reply:
done